java/src/jmri/jmrit/swing/meter/MeterFrame.java

```diff
@@ -40,7 +42,7 @@
 
         private final double multiply;
 
-        private Unit(double m) { multiply = m; }
+        Unit(double m) { multiply = m; }
```

Comment:
Just curious. Why is the constructor not private? It should never be called by anyone except the class itself. I have no problem with the change, but I want to learn if I have missed something.

Reply:
It was flagged as redundant. A nested enum is private by default (most online examples show independent, class level enums where private is not the default, but the difference is seldom obvious).

Reply:
Thanks.